### DISCUSSION TITLE
Move `@aws-sdk/client-sqs` from `dependencies` to `devDependencies`

### DISCUSSION THREAD

**Discussion by ruscon** [bbc/sqs-consumer#435] [Feature Request]
### Background

Because `@aws-sdk/client-sqs` is already indicated as `peerDependencies`, there is no sense in having it in `dependencies`

```json
{
  "dependencies": {
    "@aws-sdk/client-sqs": "^3.428.0",
    "debug": "^4.3.4"
  },
  "peerDependencies": {
    "@aws-sdk/client-sqs": "^3.428.0"
  }
}
```

### Objectives

I think it's worth moving to `devDependencies`.

**Comment by nicholasgriffintn** [comment]
Between 4-7, NPM stopped auto installing peerDependencies, so that is why it is in both.

The dependency can probably be removed now though, happy to take a PR for it. I
